pkgs/development/python-modules/nanobind/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, lib, buildPythonPackage, pythonOlder, fetchFromGitHub, cmake, eigen
+, python, pytestCheckHook, numpy, scipy, torch, jax, jaxlib
+, tensorflow
+}:
+buildPythonPackage rec {
+  pname = "nanobind";
+  version = "1.8.0";
+  format = "setuptools";
+
+
+  src = fetchFromGitHub {
+    owner = "wjakob";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-MkbdjUL9ZC/JVBCSCgNLqTObISczPhJtDKKOSa85gmA=";
+    fetchSubmodules = true;
+  };
+
+  disabled = pythonOlder "3.8";
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ eigen ];
+  dontUseCmakeBuildDir = true;
+
+  postBuild = ''
+    # build tests
+    make -j $NIX_BUILD_CORES
+  '';
+
+  # FIXME Cannot import tensorflow and torch simultaneously ("Found duplicated packages in closure for dependency 'tensorboard'")
+  nativeCheckInputs = [
+    pytestCheckHook
+    numpy
+    scipy
+    torch
+    # tensorflow
+    jax
+    jaxlib
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/wjakob/nanobind";
+    changelog =
+      "https://github.com/wjakob/nanobind/blob/${src.rev}/docs/changelog.rst";
+    description = "tiny and efficient C++/Python bindings";
```

Comment:
```suggestion
    description = "Tiny and efficient C++/Python bindings";
```

Reply:
@dotlambda thanks for you incredibly quick review - highly appreciated